App/lib/hooks/useI18n.js

```diff
@@ -0,0 +1,17 @@
+import { useSelector } from 'react-redux';
+import get from 'lodash/get.js';
+
+import { selectors } from 'store';
+import defaultTranslations from 'translations';
+
+export default function useI18n(args) {
+  const defaultLanguage = useSelector(selectors.language);
+  const language = args?.language || defaultLanguage;
+  const translations = args?.translations || defaultTranslations;
+
+  function i18n(key) {
+    return get(translations, `[${language}][${key}]`, 'xxx');
```

Comment:
It might be good to return the `key` variable as a fallback here instead of `xxx` so we know which key is faulty.

Reply:
That's a good idea, I don't like xxx, but wasn't sore what to return...